File: 01-OopsIntro/Employee.java
package OopsIntro;

/**
 * Employee
 */
public class Employee {

    private int salary; // to store the salary of employee
    
    public String employeeName; // to store the name of employee
    
    // Method to set the employee name as given input
    public void setName(String s) {
        employeeName = s;
    }
    
    // Method to set the salary as given input
    public void setSalary(int val) {
        salary = val;
    }
    
    // Method to get the salary of the employee
    public int getSalary() {
        return salary;
    }
}

File: 01-OopsIntro/Employee.py
class Employee:
    def __int__(self):
        self.__salary=0
        self.employeeName = ""

    def setSalary(self,sal):
        self.__salary = sal

    def setempname(self,name):
        self.employeeName=name

    def getempname(self):
        return self.employeeName
if __name__ == "__main__":
    obj = Employee()
    obj.setempname("Nilesh")
    print(obj.getempname())



